test/integration/targets/azure_rm_functionapp/tasks/main.yml

```diff
@@ -1,14 +1,19 @@
+- name: Prepare random number
+  set_fact:
+    rpfx: "{{ resource_group | hash('md5') | truncate(7, True, '') }}{{ 1000 | random }}"
+  run_once: yes
+
 - name: create storage account for function apps
   azure_rm_storageaccount:
     resource_group: '{{ resource_group }}'
-    name: azfunccistor4
+    name: azfunccistor4{{ rpfx }}
```

Comment:
Use the `resource_prefix` variable for a value which is unique to the test run instead of generating one.

This should work:

```yaml
    name: "{{ resource_prefix }}"
```

It is generated by ansible-test on each test run and contains some identifying information about the test as well as a unique value to differentiate it from previous test runs.

Reply:
If multiple values are needed, use it as a prefix and append an additional value.

Reply:
ok, i will do that. where's resource_prefix coming from? is it something new? i guess i should start updating all my tests....

Reply:
It's generated by `ansible-test` and was part of the initial support for cloud testing:

https://github.com/ansible/ansible/blob/b9b8081a878c60f620deb1d4f6ed4fe1439799de/test/runner/lib/cloud/__init__.py#L362-L374

It is injected as a var when invoking `ansible-playbook` during the test run:

https://github.com/ansible/ansible/blob/b9b8081a878c60f620deb1d4f6ed4fe1439799de/test/runner/lib/cloud/azure.py#L145-L146